transportable-udfs-avro/build.gradle

```diff
@@ -1,10 +1,11 @@
 apply plugin: 'java'
 
 dependencies {
-  compile project(':transportable-udfs-api')
-  compile project(':transportable-udfs-type-system')
-  compile('org.apache.avro:avro:1.10.2')
-  testCompile project(path: ':transportable-udfs-type-system', configuration: 'tests')
+  implementation project(':transportable-udfs-api')
+  implementation project(':transportable-udfs-type-system')
+  implementation('org.apache.avro:avro:1.10.2')
+  implementation('com.google.guava:guava:24.1-jre')
```

Comment:
Looks like this PR adds more `implementation` for some dependencies (like `'com.google.guava:guava:24.1-jre'` here), I think it's caused by the difference between `compile` and `implementation`. Have we considered replacing some `implementation` by `api` to avoid adding more dependency declarations like this one? For example, I feel like that `implementation('com.google.guava:guava:19.0')` can be changed to `api` in `transportable-udfs-type-system/build.gradle`.

Reply:
Done